src/pages/Home/home.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import './home.css';
+import AddBook from '../AddBook/addbook';
+
+const Home = () => {
+  const [lists, setLists] = useState([]);
+
+  return (
+    <>
+      <div>
+        <p>ADD NEW BOOK</p>
+        <AddBook setList={setLists} />
+      </div>
```

Comment:
- [ ] Please let's follow this project structure: 
- 1. We need to have at least two-page components: `<Books>` and `<Categories>` (<Error>, optional)
2. A `<Books>` component, should contain or import the following components:
   * `<BookList>`
     * Renders a list of books using the `<Book>` component
   * `<Book>`
     * Takes the book data as a prop and renders the title and the author. It should have a button for removing books
   * `<NewBookForm>` or Addbook component 
     * Renders a form with inputs for the title and the author of the book. It should have a button to submit.
   
- Note: The <BookList>`, <Book>` and <AddBookForm> should be created inside the Components directory. Please make this 👍🏻